packages/cli/src/util/createServer.ts

```diff
@@ -149,6 +150,7 @@ type CreateBaseServerOptions = {
   multiprocess: boolean;
   errors: boolean;
   verboseLevel: string;
+  jsonSchemaFakerFillProperties: boolean;
```

Comment:
I'm not sure if this is the right place, but I don't see a better opportunity to add a JS Doc comment that describes what effect this option is intended to have.

Reply:
added a comment in the most recent changes